src/main/java/jenkins/branch/ItemColumn.java

```diff
@@ -65,17 +64,26 @@ public boolean isPrimary(Object job) {
     }
 
     /**
-     * Gets the description of a job.
+     * Gets the tool-tip title of a job.
      *
      * @param job the job.
-     * @return the description.
+     * @return the tool-tip title unescaped for use in an attribute.
      */
     @SuppressWarnings("unused") // used via Jelly EL binding
-    public String getDescription(Object job) {
+    public String getTitle(Object job) {
```

Comment:
It is a part of the 2.0 release, hence the compatibility breakage is acceptable. Right?